src/main/resources/template/scienceEuropeTemplate.resource

```diff
@@ -3,6 +3,9 @@ metadata.no=As there are no domain specific metadata standards applicable, we wi
 
 dataOrganisation.no=
 
+dataQualityControl.avail=The following data quality checks will be done:
+dataQualityControl.no=Data quality checks will be done, e.g. checks of consistency of labels, logical errors in the data, data curation, and version control.
```

Comment:
Should this be "No data quality checks will be done...."?

Reply:
renamed to dataQualityControl.default to better represent its new functionality